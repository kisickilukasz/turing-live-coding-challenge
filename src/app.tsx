import './app.css';
import { IteratorTest } from './components/IteratorTest';

export function App() {
    return (
        <IteratorTest />
    );
}
