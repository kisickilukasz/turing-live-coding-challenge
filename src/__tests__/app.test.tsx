import {screen, render} from '@testing-library/react';

import { App } from '../app';

test('renders app component', () => {
  const { container } = render(<App />);

  expect(screen.getByTestId('iterator-container')).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
