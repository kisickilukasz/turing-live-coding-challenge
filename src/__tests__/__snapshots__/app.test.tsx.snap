// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app component 1`] = `
<div>
  <div
    data-testid="iterator-container"
  >
    <p>
      All users:
       
    </p>
    <div>
      Current user:
       
    </div>
    <div>
      Users could not be fetched!!
    </div>
    <button
      style="background: grey; color: white; padding: 20px;"
    >
      Next
    </button>
    <button
      style="background: grey; color: blue; padding: 20px;"
    >
      Previous
    </button>
  </div>
</div>
`;
